frontend/src/components/Modals/MangeWorkspace/Settings/index.jsx

```diff
@@ -140,6 +140,40 @@ export default function WorkspaceSettings({ workspace }) {
                 onChange={() => setHasChanges(true)}
               />
             </div>
+
+            <div>
+              <div className="flex flex-col gap-y-1 mb-4">
+                <label
+                  htmlFor="name"
+                  className="block text-sm font-medium text-gray-900 dark:text-white"
+                >
+                  Chat History
+                </label>
+                <p className="text-xs text-gray-600 dark:text-stone-400">
+                  Chat history: how far back in teh current discussion will 
+                  chat pull responses.
+                  <br />
+                  Default 20. Anything more than 45 is likely to cause significant failures
+                  as well as higher costs.
+                  <br />
+                  Recommended: 20
+                </p>
+              </div>
+              <input
+                name="openAiHistory"
+                type="number"
+                min={0.0}
+                max={45.0}
```

Comment:
These can be whole numbers since step is 1! Not a problem if left though

Reply:
updated to ints